src/NodeVisitor/UseNamespaceScoperNodeVisitor.php

```diff
@@ -22,6 +22,9 @@ public function __construct($prefix)
     public function enterNode(Node $node)
     {
         if ($node instanceof UseUse) {
+            if ($node->hasAttribute('phpscoper_ignore')) {
```

Comment:
I've left typehinting separate for now. I'll add issues if I don't swing past these and the Slack mentioned items tomorrow evening.